### DISCUSSION TITLE
SH-RPi v2 development update

### DISCUSSION THREAD

**Discussion by mairas** [hatlabs/discussions#42] [Show and tell]
Hi all,

I've been busy developing Sailor Hat for Raspberry Pi v2 and wrote a small update post on the topic. Check it out here:

https://hatlabs.fi/post/2022/12/25/shrpi-v2-update.html

**Comment by DangarStu** [comment]
Thanks for the update Matti, exiciting to see things getting so close to being ready!

**Comment by MarcusHjortsberg** [comment]
Quite interested in the SH-RPI v2! Would it be possible to pre-order one? Thx

**Reply by mairas** [reply]
Hi,

I haven't really been doing pre-orders, at least for unfinished products. I'll be sure to announce the release here, and the initial batch will be big enough so they won't run out. 😊

The current situation is that the design is ready and tested, but the last proto batch still had an issue with the GPIO connector footprint (the factory botched them, basically). I need to order one more proto batch to verify the fix but haven't been able to do that due to the Chinese New Year. As soon as I have verified the fix with the new protos, I'll be placing the production order.
